spec/outputs/elasticsearch_spec.rb

```diff
@@ -486,8 +486,13 @@ def mock_actions_with_response(*resp)
         .any_instance.stub(:bulk).and_return(*resp)
       LogStash::Outputs::Elasticsearch::Protocols::NodeClient
         .any_instance.stub(:bulk).and_return(*resp)
-      LogStash::Outputs::Elasticsearch::Protocols::TransportClient
-        .any_instance.stub(:bulk).and_return(*resp)
+    end
+
+    def unmock_actions()
```

Comment:
I see. so then my issue is probably that I was stubbing out both the TransportClient and NodeClient. I believe something was conflicting. That is why the TransportClient stubbing is deleted